bin/save-version.sh

```diff
@@ -17,7 +17,7 @@
 # Note: for internal (aka pre-release) versions, the version should have
 # "-INTERNAL" appended. Parts of the code will look for this to distinguish
 # between released and internal versions.
-VERSION=2.2.0-hdp2.3.0-rocana1.4.0-rc1
+VERSION=2.2.0-hdp2.3.0-rocana1.5.0-rc1
```

Comment:
Question because I don't know the answer: Is this `rc1` for the Impala release, or should it be `rc2` because it's for `rocana-1.5.0-rc2`?